src/App.tsx

```diff
@@ -1,14 +1,57 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
+// import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+function filterTodos(todos: Todo[], todosActivityFilter: string, todosQuery: string) {
+  let resultTodos = [...todos];
+  const query = todosQuery.toLowerCase();
+
+  switch (todosActivityFilter) {
+    case 'Completed':
```

Comment:
```
export enum SelectType {
  All = 'all',
  Active = 'active',
  Completed = 'completed',
}
```

you can create an enam type for Select options and use it like this, bonus - tips 
`case SelectType.Active:`